sarazaToussaint/Way_Farer
# #167279280 ft(delete):Add user delete booking option [Start #167279280]

#### What does this PR do?
Have delete booking option
#### Description of Task to be completed?
Have one task to be competed by user
which is to delete his/her booking in way farer system
#### How should this be manually tested?
User should open the bookings list first
then he can select his/her needed booking and click delete button
the booking should be successfully deleted
#### Any background context you want to provide?
Design of font end with no framework
#### What are the relevant pivotal tracker stories?
#167279280
